### PR TITLE
Add files to new branch for review

### PR DESCRIPTION
**Pull request by Zuwairanajma**
Here I have:
-Created a new repo.
-Copied all my files from admission trials.
-Created a new branch for the review of codes.

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,21 @@
+nav {
```
Comment (Zuwairanajma) [inline css/style.css:1]: I suggest you always add a predefined style for your body element before you start inputting your main style. Try setting your body element to padding and margin of zero and box-sizing of border-box.

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    width: 20%;
+    height: 10%;
+    box-sizing: border-box;
+  }
+  
+  @media (max-width: 768px) {
+   html {
+     font-size: 78.5%;
+   } 
+  }
```
Comment (Zuwairanajma) [inline css/style.css:21]: Good job using the right spacing and newlines for your stylesheet codes. However, as a result of not having linters in your work you were adapting the 4 spacing indentation method which is currently not the standard, I advise you install the necessary linters to your code editors.

File: index.html [hunk 3]
```diff
@@ -9,18 +9,27 @@
...
+        <a class='nav-link' href='#about'>About</a>
+        <a class='nav-link' href='#product'>Product</a>
+      </nav>
+    </header>
+    <section id='home'>
+      This is the home section
+    </section>
+    <section id='about'>
+      This is the about section
+    </section>
```
Comment (Zuwairanajma) [inline index.html:25]: Consider giving your text a  class name, and a tag name such as paragraph tags or other tags. for readability, easy styling, and maintenance.

**Review by Zuwairanajma — COMMENTED** [review]
Changes Required♻️
While you have put much effort into this project, I strongly suggest you document your repository in a professional way adding the GitHub workflow and the linters file.
Please try using a professional Readme.md file. I suggest you use the Microverse template to get a befitting Readme.md file.
Check my inline comment to implement other changes.

Happy Coding!

**Review by wayungi — APPROVED** [review]
Hi @Zuwairanajma , 

image

Great work, your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**Remember: Every finish line 🏁  is the beginning of a new race 🐎** 

## Highlights
- [ ] GitHub inline features used ✔️ 
- [ ] Comments are clear with reasons for change given✔️ 
- [ ] At least 2 comments given ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Zuwairanajma — COMMENTED** [review]
While you have put much effort into this project, There is still more to do to get this project approved. 
Your Readme.md file needs to be professional, try using the Microverse Readme.md template to get a better markdown file. Add a link to your license in your Readme.md File.
Your project has no linter file in it, Try adding the linters file and install the necessary extensions for smooth and cleaner job.

**Pull request merged**